apps/legacy/src/pages/ManageTokens/AddToken.tsx

```diff
@@ -61,11 +61,17 @@ export function AddToken() {
         address: addressInput,
       });
       history.goBack();
-    } catch (_err) {
+    } catch (_err: any) {
       capture('ManageTokensAddCustomToken', {
         status: 'failed',
         address: addressInput,
       });
+      if (_err.includes('already exists')) {
```

Comment:
This will throw a `TypeError` if `_err` is not a `string`.

Let's type `_err` as `unknown` and add a proper typecheck, i.e. `if (typeof _err === 'string')`.